### PR TITLE
Readd the crawl comment on PRs

### PR DESCRIPTION
**Pull request by Molter73**
This is a useful comment that states how many packages and bundles have been created by a CI run. It also prints the path to the bucket everything is stored in for ease of debugging.

### CODE REVIEW

File: .github/workflows/pr-comment.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        run: |
+          IFS=',' read -r -a package_buckets <<< "$KERNEL_PACKAGE_BUCKET"
+          IFS=',' read -r -a bundle_buckets <<< "$KERNEL_BUNDLE_BUCKET"
+
+          export KERNEL_PACKAGES_STAGING_BUCKET
+          export KERNEL_BUNDLES_STAGING_BUCKET
+          export LAST_UPDATED
+          KERNEL_PACKAGES_STAGING_BUCKET="${package_buckets[0]}"
+          KERNEL_BUNDLES_STAGING_BUCKET="${bundle_buckets[0]}"
+          LAST_UPDATED="$(date)"
```
Comment (Stringy) [inline .github/workflows/pr-comment.yml:45]: [ultranit] can't we inline these?

```suggestion
          export KERNEL_PACKAGES_STAGING_BUCKET="${package_buckets[0]}"
          export KERNEL_BUNDLES_STAGING_BUCKET="${bundle_buckets[0]}"
          export LAST_UPDATED="$(date)"
```

**Review by Stringy — APPROVED** [review]
Some very minor questions/comments but nothing to prevent a merge. looks good!

**Comment by Stringy** [comment]
I also just noticed that the buckets aren't populated correctly in the comment because the env vars are slightly wrong

`KERNEL_PACKAGES_STAGING_BUCKET` -> `KERNEL_PACKAGE_STAGING_BUCKET`
`KERNEL_BUNDLES_STAGING_BUCKET` -> `KERNEL_BUNDLE_STAGING_BUCKET`

File: .github/workflows/crawl.yml [hunk 2]
```diff
@@ -57,6 +65,12 @@ jobs:
           make sync
           git --no-pager diff kernel-package-lists
 
+      - name: Uploaded packages
+        id: uploaded-packages
+        run: |
+          IFS=',' read -r -a bucket_names <<< "$KERNEL_PACKAGE_BUCKET"
+          echo "uploaded-packages=$(gsutil ls "${bucket_names[0]}" | wc -l | xargs)" >> "${GITHUB_OUTPUT}"
```
[the remark this answers is not in the document]
Comment (Molter73) [reply .github/workflows/crawl.yml:72]: It catches the error when the bucket doesn't exist and turns it into `0`, I copy pasted it from the original workflow and was as confused by it as you might be.

File: .github/workflows/pr-comment.yml (hunk 1 above)
Comment (Molter73) [reply .github/workflows/pr-comment.yml:45]: We could, but actionlint complains that shellcheck complains that variables should be exported and assigned separately 🤷🏻‍♂️

**Comment by Molter73** [comment]
> I also just noticed that the buckets aren't populated correctly in the comment because the env vars are slightly wrong
> 
> `KERNEL_PACKAGES_STAGING_BUCKET` -> `KERNEL_PACKAGE_STAGING_BUCKET` `KERNEL_BUNDLES_STAGING_BUCKET` -> `KERNEL_BUNDLE_STAGING_BUCKET`

🤦🏻‍♂️ let me try that again.

File: kernel-crawler/kernel-crawler.py [hunk 3]
```diff
@@ -591,6 +591,7 @@
             "subdirs": [
                 "OL7/developer_UEKR5/x86_64/",
                 "OL7/UEKR6/x86_64/",
+                "OL8/UEKR7/x86_64/",
```
Comment (Molter73) [inline kernel-crawler/kernel-crawler.py:594]: - [ ] Revert this change before merging.
